app/controllers/html_player_controller.rb

```diff
@@ -4,7 +4,6 @@ class HtmlPlayerController < ApplicationController
   layout 'simple'
 
   def show
-    @slide.increment(:page_view).increment(:total_view).save
-    @slide.increment(:page_view).increment(:embedded_view).save
+    @slide.increment!(:page_view).increment!(:total_view).increment!(:embedded_view)
```

Comment:
__[RuboCop]__

Rails/SkipsModelValidations: Avoid using `increment!` because it skips validations.


<!-- Comment by [SideCI](https://sideci.com) -->